source/elements/oneMKL/source/domains/dft/config_params/data_layouts.rst

```diff
@@ -0,0 +1,172 @@
+.. SPDX-FileCopyrightText: 2019-2020 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_dft_config_data_layouts:
+
+Configuration of Data Layouts
+-----------------------------
+
+The DFT interface provides the configuration parameters
+``config_param::FWD_STRIDES`` (resp. ``config_param::BWD_STRIDES``) and
+``config_param::FWD_DISTANCE`` (resp. ``config_param::BWD_DISTANCE``) to define
+the data layouts used in forward (resp. backward) domain. Considering the
+:math:`m`-th (:math:`m \geq 0`) :math:`d`-dimensional
+:math:`n_{1}\times n_{2} \times\dots\times n_d` data set :math:`X` in forward
+(resp. backward) domain, those configuration parameters describe that a
+particular element :math:`X^{m}_{k_{1}, k_{2}, \dots, k_d }` is located at index
+
+.. math::
+    s_0 + k_1*s_1 + k_2*s_2 + \dots + k_d*s_d + m*l
+    :label: eq_idx_data_layout
+
+of the corresponding data container (``sycl::buffer`` or USM pointer) provided
+to the compute function, the data type of which is (possibly implicitly
+re-interpreted) as documented in the
+:ref:`table<onemkl_dft_config_data_element_types>` below. In the index
+expression :eq:`eq_idx_data_layout`,
+
+- :math:`s_j`, :math:`0 \leq j \leq d` represents the :ref:`generalized strides and offset<onemkl_dft_fwd_bwd_strides>` within :math:`d`-dimensional data sets in forward (resp. backward) domain, in number of elements of the :ref:`relevant elementary data type<onemkl_dft_config_data_element_types>`;
+
+- :math:`l` represents the :ref:`distance<onemkl_dft_fwd_bwd_distance>` between successive :math:`d`-dimensional data sets in forward (resp. backward) domain, in number of elements of the :ref:`relevant elementary data type<onemkl_dft_config_data_element_types>`;
+
+.. container:: section
+
+   .. _onemkl_dft_config_data_element_types:
+
+   .. rubric:: Elementary data type assumed by descriptors depending on their type and configuration (precision matching the :ref:`descriptor's<onemkl_dft_enum_precision>`)
```

Comment:
In the html file this isn't really formatted like a header, and it looks a little weird to me as a sentence fragment. Maybe a colon at the end? Or a full sentence like "The following table lists the elementary data type..."?

Reply:
Changed it entirely, please let me know if you're ok with it (thumbs-up) or comment.